src/thermo/ThermoFactory.cpp

```diff
@@ -88,6 +89,8 @@ ThermoFactory::ThermoFactory()
     addAlias("liquid-water-IAPWS95", "PureLiquidWater");
     reg("binary-solution-tabulated", []() { return new BinarySolutionTabulatedThermo(); });
     addAlias("binary-solution-tabulated", "BinarySolutionTabulatedThermo");
+    reg("Peng-Robinson", []() { return new PengRobinson(); });
+    addAlias("Peng-Robinson", "PengRobinson");
```

Comment:
I don't think we need the alias here, since we're not adding the XML input.

Reply:
Removed the alias.